package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -1123,13 +1220,25 @@ class NCDFPicklable(scipy.io.netcdf.netcdf_file):
     This means that for a successful unpickle, the original file still has to
     be accessible with its filename.
 
+
+    .. note::
+        This class subclasses :class:`scipy.io.netcdf.netcdf_file`, please
+        see the `scipy netcdf API documentation`_ for more information on
+        the parameters and how the class behaviour.
+
+
     Parameters
     ----------
     filename : str or file-like
         a filename given a text or byte string.
     mmap : None or bool, optional
         Whether to mmap `filename` when reading. True when `filename`
         is a file name, False when `filename` is a file-like object.
+    version : {1, 2}, optional
+        Sets the netcdf file version to read / write. 1 is classic, 2 is
+        65-bit offset format. Default is 1.
```

Comment:
```suggestion
        64-bit offset format. Default is 1 (but note AMBER ncdf *requires* 2).
```

Or change the default to 2, it's AMBER reader after all.

Reply:
Oops missed that thanks!. Personally I prefer we keep true to the original scipy class - just in case we end up getting confused in the future (there are other MD engines / workflows that use netcdf out there [see Yank] that don't necessarily stick to the way AMBER does things [not checked that version_byte yank uses]).